ansible/roles/filebeat/tasks/main.yml

```diff
@@ -0,0 +1,61 @@
+---
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,e
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# Installing Filebeat
+#
+
+# download Filebeat
+- name: "download filebeat rpm"
+  get_url:
+  args:
+    url: https://artifacts.elastic.co/downloads/beats/filebeat/{{ filebeat_rpm }}
+    dest: /tmp/{{ filebeat_rpm }}
+    checksum: "{{ filebeat_sha512 }}"
+    force: no
+
+  #Install filebeat"
+- name: "Install filebeat rpm"
+  become: true
+  yum:
+    name: /tmp/{{ filebeat_rpm }}
+    state: present
+
+#Configure Filebeat yml file
+
+- name: Copy filebeat.yml onto ec2
+  copy:
```

Comment:
I think you should consider using `template`, rather than `copy` for tasks like this where we want to expand Ansible variables. For example, within filebeat.yml, I observed hard-coded directory paths, which should be templatized to use `worker_data_dirs`. Subsequently, this Ansible task itself would require to become a `template` to allow expansion.

Reply:
also, the reference to `ec2` in this task name seems incorrect. perhaps you really meant `etc` instead of ec2?

Reply:
In the interest of helping close things, I have addressed these in a [commit on my own fork](https://github.com/arvindshmicrosoft/fluo-muchos/commit/d2d6b2d20c36052796de5b908ca7bb2d477f9e4b) - could you please take a look at that, merge as appropriate and then push changes to this PR?